src/data/reformDefinitionCode.js

```diff
@@ -43,89 +41,36 @@ export function getHeaderCode(type, metadata, policy) {
+    Object.keys(policy.baseline.data).length === 0
+  ) {
     return [];
   }
-
-  // Calculate the earliest start date and latest end date for
-  // the policies included in the simulation
-  const { earliestStart, latestEnd } = getStartEndDates(policy);
-
-  return [
-    "",
-    "",
-    `"""`,
-    "In US nationwide simulations,",
-    "use reported state income tax liabilities",
-    `"""`,
-    "def use_reported_state_income_tax(parameters):",
-    "    parameters.simulation.reported_state_income_tax.update(",
-    `        start=instant("${earliestStart}"), stop=instant("${latestEnd}"),`,
-    "        value=True)",
-    "    return parameters",
-    "",
-    "",
-    "class baseline_reform(Reform):",
-    "    def apply(self):",
-    "        self.modify_parameters(use_reported_state_income_tax)",
-  ];
+  let json_str = JSON.stringify(policy.baseline.data, null, 2);
+  let lines = [""].concat(json_str.split("\n"));
+  lines[1] = "baseline = Reform.from_dict({" + lines[0];
+  lines[lines.length - 1] =
+    lines[lines.length - 1] + ', country_id="' + metadata.countryId + '")';
+  return lines;
 }
 
-export function getReformCode(type, policy, region) {
-  // Return no reform code for households or for policies
-  // without reform parameters
-  if (Object.keys(policy.reform.data).length <= 0) {
+export function getReformCode(policy, metadata) {
+  if (!policy?.baseline?.data || Object.keys(policy.reform.data).length === 0) {
     return [];
   }
-
-  let lines = ["", "", "def reform_parameters(parameters):"];
-
-  for (let [parameterName, parameter] of Object.entries(policy.reform.data)) {
-    for (let [instant, value] of Object.entries(parameter)) {
-      const [start, end] = instant.split(".");
-      if (value === false) {
-        value = "False";
-      } else if (value === true) {
-        value = "True";
-      }
-      // If param name contains number, transform into valid Python
-      if (doesParamNameContainNumber(parameterName)) {
-        parameterName = transformNumberedParamName(parameterName);
-      }
-      lines.push(
-        `    parameters.${parameterName}.update(`,
-        `        start=instant("${start}"), stop=instant("${end}"),`,
-        `        value=${value})`,
-      );
-    }
-  }
-  lines.push("    return parameters");
-
-  lines = lines.concat([
-    "",
-    "",
-    "class reform(Reform):",
-    "    def apply(self):",
-    "        self.modify_parameters(reform_parameters)",
-  ]);
-
-  // For US reforms, when calculated society-wide, add reported state income tax
-  if (type === "policy" && US_REGIONS.includes(region)) {
-    lines.push("        self.modify_parameters(use_reported_state_income_tax)");
-  }
-
+  let json_str = JSON.stringify(policy.reform.data, null, 2);
+  let lines = [""].concat(json_str.split("\n"));
+  lines[1] = "reform = Reform.from_dict({" + lines[0];
+  lines[lines.length - 1] =
+    lines[lines.length - 1] + ', country_id="' + metadata.countryId + '")';
```

Comment:
Does this piece of code need to reformat policies with numbers as keys (e.g., the US income tax brackets)? At the moment, this doesn't sanitize, causing one of the test failures we're seeing, but I'm not sure if it needs to.

Reply:
We don't need to- could you remove the test?

Reply:
Can do